tests/sles4sap/redirection_tests/ensa2_kill_sapinstance.pm

```diff
@@ -0,0 +1,116 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary: Test module performs ENSA2 B<'Kill sapinstance'> test scenario.
+
+use parent 'sles4sap::sap_deployment_automation_framework::basetest';
+
+use warnings;
+use strict;
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use utils qw(script_retry);
+use hacluster;
+use sles4sap::sap_host_agent qw(saphostctrl_list_instances);
+use sles4sap::sapcontrol;
+use sles4sap::console_redirection;
+
+=head1 SYNOPSIS
+
+Test module performs ENSA2 B<'Kill sapinstance'> test scenario. SAP instance process (ERS or ASCS) B<*.sap*> is killed
+using C<kill -9> command.
+Test module is based on usage of console redirection. Check B<tests/sles4sap/redirection_tests/README.md> for details.
+
+Test module requires following structure containing information:
+    $run_args->{scenarios} = [
+        'GottaCatchEmAll' => {
+            description => 'Test throws pokeball at Pikachu and captures it.',
+            connect_ip => 192.168.1.1,
+            connect_user => Ash,
+            resource_name => pokeball
```

Comment:
Can you provide a list of example valid values for `resource_name` or a reference to where a list of them is defined?

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/21954/files#diff-52989d4148425f8fb26d868fbaa279b711ca972100069c01494fa97ce8999ad0R48

Reply:
Yep. I updated the whole thing with normal examples.
I as well changed argument `resource_name` to `crm_resource_name`.